_pages/extension_sharing_strategies.md

```diff
@@ -0,0 +1,143 @@
+      * **src: <URL>** : URL to the public repository with the sources of the extension
+      * **pip: <URL>** : URL for installing the extensions from PyPi
+   * **/README.(md|rst)** : REQUIRED readme with basic description of the extension, badges for CI, install instructions etc.
+   * **/review**: OPTIONAL folder with RST documents related to the review of the extension for acceptance as part of the NWB:N core schema. If present the /review folder MUST contain:
+      * **/review/Makefile** : Standard Sphinx makefile for building the review docs  in HTML, LATEX etc.
+      * **/review/make.bat** : Standard Sphinx make.bat file
+      * **/review/README.md** : Standard readme file with instructions on how to build and use the docs .
+      * **/review/source** : Folder with the RST sources for the Sphinx review documents
+         * The /review/source folder MUST contain:
+            * **/docs/source/conf.py**: Sphinx configuration file for building the docs.
+            * **/docs/source/index.rst** : Main RST file for the review docs.
+            * **/docs/source/release_notes.rst** : RST file with the release notes for the review docs.  This file MUST be included index.rst.
+            * **/docs/source/review.rst** : Main document with the review discussion for the NDX. This file MUST be included in index.rst
+         * The /review/source folder MAY contain the following OPTIONAL documentation files:
+            * **/docs/source/credits.rst** : RST file with the credits for the extension review. This file MUST be included (directly or indirectly) in the main index.rst file.
+            * **/docs/source/legal.rst** : RST file with the legal details about the extension review (e.g., licence and copyright). This file MUST be included (directly or indirectly) in the main index.rst file.
+         * The /review/source folder MAY contain the following OPTIONAL folders:
+            * **/review/source/_static** : Static files (e.g., images) used by the documentation MUST be stored in this OPTIONAL folder.
+   * **/.gitignore**: Containing standard ignore options to ignore sphinx build files, autogenerated docs files from the nwb-docutils and sphinx, python and matlab temporary files, etc.
+
+
+#### 2.3.2 ndx-custom: Custom NWB:N Extensions
+The ndx-custom repository contains the sources of the user extension that has usually been created from the ndx-template. This repository lives in the users own Git space and is NOT part of the nwb-extensions organization.
+
+### 2.4 Additional Tools and Methods
+
+#### 2.4.1 Documenting extensions
+
+The [nwb-docutils](https://github.com/NeurodataWithoutBorders/nwb-docutils) provide tools for generating Sphinx RST documentation directly from the YAML sources. The “generate_format_docs.py” command-line tools MUST be used to auto-generate documentation from the YAML sources.
```

Comment:
Is this for the review docs? As I mentioned earlier, I think we could probably just link a few markdown files together and call it a day

Reply:
The nwb-docutils are for creating the documentation for the extenions itself (not for the review docs). This is equivalent to https://nwb-schema.readthedocs.io/en/latest/